src/stravavis/cli.py

```diff
@@ -28,15 +34,20 @@ def main():
 
     # Normally imports go at the top, but scientific libraries can be slow to import
     # so let's validate arguments first
+    from stravavis.plot_calendar import plot_calendar
     from stravavis.plot_elevations import plot_elevations
     from stravavis.plot_facets import plot_facets
     from stravavis.plot_landscape import plot_landscape
     from stravavis.plot_map import plot_map
+    from stravavis.process_activities import process_activities
     from stravavis.process_data import process_data
 
     print("Processing data...")
     df = process_data(args.path)
 
+    print("Processing activities...")
+    activities = process_activities(activities_path)
```

Comment:
There's no `activities_path` here, we need to pass `args.activities_path`:

```suggestion
    activities = process_activities(args.activities_path)
```